### PR TITLE
img/@title; https://github.com/metanorma/metanorma-standoc/issues/135

### PR DESCRIPTION
**Pull request by opoudjis**

### CODE REVIEW

File: author/topics/document-format/text.adoc [hunk 1]
```diff
@@ -274,6 +274,32 @@ from the image.
 image::logo.jpg
 --
 
+=== Captions and titles
+
+As elsewhere in Metanorma, the caption of an image (in Metanorma, of the figure containing the image)
+is given with a line prefixed with dot. In Asciidoctor, the title attribute is treated as identical
+to the dot-prefixed caption:
+
+[source,asciidoctor]
```
Comment (ronaldtse) [inline author/topics/document-format/text.adoc]: There is `[source,adoc]` but not `[source,asciidoctor]`. Please help update (all other instances have been fixed).

File: author/topics/document-format/text.adoc [hunk 2]
```diff
@@ -274,6 +274,32 @@ from the image.
...
+image::logo.jpg[title=Caption]
+--
+
+Metanorma supports a title attribute on images for accessibility, which is distinct from the figure
+caption. This is entered in Metanorma as a `titleattr` attribute:
+
+[source,asciidoctor]
+--
+[titleattr=Title Attribute]
+image::logo.jpg
```
Comment (ronaldtse) [inline author/topics/document-format/text.adoc]: And this fails?

```
image::logo.jpg[titleattr="Accessiblity Title"]
```

Comment (opoudjis) [reply author/topics/document-format/text.adoc]: Gem not released yet, obviously. It works for me locally.

Why would you assume something I have coded yesterday is available in production today?!

File: author/topics/document-format/text.adoc (hunk 1 above)
Comment (opoudjis) [reply author/topics/document-format/text.adoc]: Updated PR to change all instances

**Pull request merged**